frog/imports/client/Wiki/index.js

```diff
@@ -520,6 +595,82 @@ class WikiComp extends React.Component<WikiCompPropsT, WikiCompStateT> {
     return { liId, pageId };
   };
 
+  editAccess = async action => {
```

Comment:
What does this function do?

Reply:
I will add a JSDoc comment explaining the function